packages/frontend/src/pages/Send/Send.tsx

```diff
@@ -112,6 +112,48 @@ const Send: FC = () => {
     _setToNetwork(_toNetwork)
   }, [queryParams, networks])
 
+  // use the values saved to localStorage for default network selection (if they exist)
+  useEffect(() => {
+    const fromNetworkString = localStorage.getItem('fromNetwork')
+    const savedFromNetwork = fromNetworkString ? JSON.parse(fromNetworkString) : ''
+
+    const toNetworkString = localStorage.getItem('toNetwork')
+    const savedToNetwork = toNetworkString ? JSON.parse(toNetworkString) : ''
+
+    if (savedFromNetwork) {
+      setFromNetwork(savedFromNetwork)
+    } else if (!queryParams.sourceNetwork) {
+      setFromNetwork(networks.find(network => network.chainId === 1)) // use mainnet if no default origin exists
```

Comment:
suggestion: check for ethereum chain id on both mainnet and goerli

```suggestion
setFromNetwork(networks.find(network => network.chainId === networkSlugToId(ChainSlug.Ethereum)))
```